src/FlexView.tsx

```diff
@@ -176,47 +165,48 @@ export class FlexView extends React.Component<FlexView.Props> {
       'marginRight',
       'marginBottom'
     ]);
-    return { ...this.getFlexStyle(), ...style, ...this.props.style };
-  }
 
-  getContentAlignmentClasses(): string {
-    const vPrefix = this.props.column ? 'justify-content-' : 'align-content-';
-    const hPrefix = this.props.column ? 'align-content-' : 'justify-content-';
-
-    const vAlignContentClasses = {
-      top: `${vPrefix}start`,
-      center: `${vPrefix}center`,
-      bottom: `${vPrefix}end`
-    };
+    function alignPropToFlex(align: FlexView.Props['vAlignContent'] | FlexView.Props['hAlignContent']) {
+      switch (align) {
+        case 'top':
+        case 'left': return 'flex-start'
+        case 'center': return 'center'
+        case 'bottom':
+        case 'right': return 'flex-end'
+      }
+    }
 
-    const hAlignContentClasses = {
-      left: `${hPrefix}start`,
-      center: `${hPrefix}center`,
-      right: `${hPrefix}end`
+    return {
+      boxSizing: 'border-box',
+
+      // some browsers don't set these by default on flex
+      minWidth: 0,
+      minHeight: 0,
+
+      // flex properties
+      display: 'flex',
+      flexDirection: column ? 'column' : 'row',
+      flexWrap: wrap ? 'wrap' : 'nowrap',
+      flex: `${this.getGrow()} ${this.getShrink()} ${this.getBasis()}`,
+      justifyContent: alignPropToFlex(column ? vAlignContent : hAlignContent),
+      alignItems: alignPropToFlex(column ? hAlignContent : vAlignContent),
+
+      // style passed throw props
+      ...style,
+      ...this.props.style
     };
-
-    const vAlignContent = this.props.vAlignContent && vAlignContentClasses[this.props.vAlignContent];
-    const hAlignContent = this.props.hAlignContent && hAlignContentClasses[this.props.hAlignContent];
-
-    return cx(vAlignContent, hAlignContent);
-  }
-
-  getClasses(): string {
-    const direction = this.props.column && 'flex-column';
-    const contentAlignment = this.getContentAlignmentClasses();
-    const wrap = this.props.wrap && 'flex-wrap';
-    return cx('react-flex-view', direction, contentAlignment, wrap, this.props.className);
   }
 
   render() {
-    const className = this.getClasses();
     const style = this.getStyle();
     const props = omit(this.props, Object.keys(FlexView.propTypes));
     return (
-      <div className={className} style={style} {...props}>
+      <div className={`react-flex-view ${this.props.className || ''}`.trim()} style={style} {...props}>
```

Comment:
Are you leaving the class to be slightly less breaking or some other reason?

Reply:
yes, I think we are relying on that `class` in some projects... for modern browsers this PR should not be breaking (apart from removing the import of the no longer existing SASS)